ui/components/multichain/network-manager/network-manager.test.tsx

```diff
@@ -9,6 +9,12 @@ import mockState from '../../../../test/data/mock-state.json';
 import { SOLANA_WALLET_SNAP_ID } from '../../../../shared/lib/accounts';
 import { NetworkManager } from './network-manager';
 
+// Mock the isFlask global
+Object.defineProperty(globalThis, 'isFlask', {
+  value: false,
+  writable: true,
+});
+
```

Comment:
Is this really required? This would """enable""" flask for all existing tests and could have some side-effects? I don't recall seeing any `isFlask` check for this PR?